src/EditorFeatures/Core/CommentSelection/AbstractToggleBlockCommentBase.cs

```diff
@@ -10,7 +10,8 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CommentSelection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection;
```

Comment:
should the items in this namespace be `Microsoft.CodeAnalysis.CommentSelection`? (and below)